pgmpy/factors/CPD.py

```diff
@@ -11,6 +12,7 @@
 from pgmpy.extern import tabulate
 from pgmpy.extern import six
 from pgmpy.extern.six.moves import range, zip
+from warnings import warn
```

Comment:
@anaviltripathi Check PEP 8 for grouping import statements: https://www.python.org/dev/peps/pep-0008/#imports